lib/cosmos/system/target.rb

```diff
@@ -167,9 +179,9 @@ def get_target_dir(path, name, gem_path)
         dir = File.join(path, name)
       end
       lib_dir = File.join(dir, 'lib')
-      Cosmos.add_to_search_path(lib_dir, false) if File.exist?(lib_dir)
+      Cosmos.add_to_search_path(lib_dir) if File.exist?(lib_dir)
```

Comment:
I specifically added a test to say we prefer our target lib folder when requiring files. I think that makes more sense then throwing it at the end of the path. Not sure why we did that. Same with 'procedures' below.